### DISCUSSION TITLE
Invest in new hardware for Goggles yes/now and if yes how (it is worth it)

### DISCUSSION THREAD

**Discussion by recom3** [recom3/api-hud-goggle#11] [Announcements]
## 👋 Welcome!
This discussion is just to write ideas, if it is worth creating a new hardware for the ski goggles. Now the original hardware snow is having ten years and still working, but:
  * New processors are faster and have smaller power consumption.
  * Displays are much better: micro-displays and even proper transparent hud´s.
  * Form factor was no so good as hardware has to be integrated inside goggles. It would be nice to split the cpu from the display, si cpu can be attached maybe to the strap (outside).
  * The original problem imo with the goggles was the lack of apps and proper market place (also the price was high). So, additionally to new hardware, these issues have to be tackled.

  To get started, comment below with an introduction of yourself (if you want) and tell us about your opinions.

**Comment by Romkabouter** [comment]
I think it would be better to see if custom software is possible. The hardware stuff will be a lot of effort and there are other options available.
Currently there is not a lot of news on the Ostloong Sirius https://ostloong.com/sirius/ 
and the rideon vision seems to be dead as well https://www.facebook.com/rideonvision/
But the Rekkie is available https://rekkie.com/products/rekkie-augmented-reality-ski-goggles and get good reviews
This has indeed a cpy outside of the goggle, but I actually like the fact that it all fits inside the goggle without some device stuck on the strap. 

In my opinion there was no original problem, except for the battery issues. I was a happy user, but when the battery dies after a couple of hours the usage becomes disappointing.
When I replaced the battery last year, it was fun using it. 

So to conclude, I think focussing on software rather than hardware has my preference. A new and more open OS or something .

**Reply by recom3** [reply]
Mostly I agree with you in the sense that the original hard was good and under valuated, so it is worth (and we are on it) updating + improving software as much as possible. Here we mean as software as the upper layer in Java UI,, but not the low level software in C++, like drivers and low level services.
In the long run the hardware will have to be updated to have more power and better displays.
Imo would be better a split system with CPU and a "pluggable" display if posible.
By the way, we have offered Rekkie guys free collaborations, maybe providing some tracking apps, but still no answer.
Their solution is using custom hardware, without Android OS, that is also gut, but makes more difficult to develop apps for them.

**Reply by recom3** [reply]
One point in favor of software argument was the failure of previous marketplace.
A good point should be to potentiate it imo.

**Reply by recom3** [reply]
Maybe a prototype with:
Board of W4100 processor + Kopin KD030308KLV-K, resolución 854x480 (micro display with prism)
could be interesting and not expensive
